content/documentation/distributed-tracing/index.adoc

```diff
@@ -18,53 +18,73 @@ toc::[]
 :icons: font
 :imagesdir: /images/documentation/tracing/
 
-Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
 
-icon:bullhorn[size=2x]{nbsp} To embed Jaeger, you must have the same protocol in Kiali and Tracing service.
+== Distributed Tracing
 
-== Kiali and Tracing Service have the same protocol
+Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io[Read docs, window="_blank"]).
 
-When Kiali and Tracing service have the same protocol, we'll have the embedded feature in the website.
+In most cases, the traces integration in Kiali should be enabled by default without any specific configuration. Kiali will look for the `istio-tracing` service in namespace `istio-system`. However if your setup differs from this default, you would have to provide a specific configuration.
 
-In the Distributed Tracing link in the menu we can see a full view for search the traces of our services.
+Note also that configuring an external URL for Jaeger will also enable more links from Kiali to Jaeger UI.
 
-image::distributed_tracing_same_protocol.png[Distributed Tracing View]
+== Configuration
 
+=== In cluster URL
 
-In each service we'll have a new tab to display the traces for the service. (If we have error traces this tab will display them)
+In order to fetch data from Jaeger, Kiali needs to get an URL that can be resolved from inside the cluster, typically using Kubernetes DNS. This is the in_cluster_url configuration. For instance, for a Jaeger service named `tracing` within `istio-system` namespace, Kiali config would be:
 
-image::service_view.png[Service View]
+```yaml
+  external_services:
+    tracing:
+      in_cluster_url: 'http://tracing.istio-system/jaeger'
+```
 
-We can interact with this embedded feature like the Jaeger UI
+icon:bullhorn[size=2x]{nbsp} If you use the Kiali operator (recommended), this config can be set in the Kiali CR. But in most cases, the Kiali operator will set a valid default in_cluster_url so you wouldn't have to change anything. If you don't use the Kiali operator, this config can be set in Kiali config map.
 
-image::trace_view.png[Trace View]
+Once the in_cluster_url setting is set on the Kiali CR, each service page will have the new Traces tab where a scatter plot will be displayed with its related traces.
```

Comment:
After looking at the result, I think finally the screenshots should be moved below (after configuration)

```suggestion
```